src/modules/dashboard/components/Whitelist/UploadAddressesWidget/UploadAddressesWidget.tsx

```diff
@@ -35,20 +36,38 @@ const MSG = defineMessages({
     id: `dashboard.Whitelist.UploadAddressesWidget.inputError`,
     defaultMessage: `TODO`,
   },
+  badFileError: {
+    id: 'dashboard.Whitelist.UploadAddressesWidget.badFileError',
+    defaultMessage: `.csv invalid or incomplete. Please ensure the file contains a single column with one address on each row.`,
+  },
+});
+
+const validationSchema = yup.object({
+  whitelistCSVUploader: yup.array().of(
+    yup.object().shape({
+      parsedData: yup
+        .array()
+        .of(yup.string())
+        .min(1, () => MSG.badFileError)
+        .max(1000, () => MSG.uploadError),
```

Comment:
How did you arrive at the max value here, since by my calculations it should be somewhere around 4200 _(42 chars in an address x 100 addresses)_

Although the min/max values don't really work, as the min value should be at least the length of an address which is 42

Reply:
Those limits are for the array of parsed data itself. So if we upload a file then the array with all of the parsed and validated addresses must be within 1 - 1000, otherwise, we throw an error.

The validation of the addresses themself are done in another file